pcdsdevices/interface.py

```diff
@@ -399,15 +399,25 @@ class MvInterface(BaseInterface):
     tab_whitelist = ["mv", "wm", "camonitor", "wm_update"]
 
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
         self._mov_ev = Event()
+        self._last_status = Status()
+        self._last_status.set_finished()
+        super().__init__(*args, **kwargs)
 
     def _log_move(self, position):
         logger.info('Moving %s from %s to %s', self.name, self.wm(), position)
 
     def _log_move_end(self):
         logger.info('%s reached position %s', self.name, self.wm())
 
+    def move(self, position, timeout=None, **kwargs):
```

Comment:
```suggestion
    def move(self, *args, **kwargs):
```

Test is breaking due to naming arguments here.